Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -76,23 +89,29 @@ public class BrokerageTransactionHandler : ITransactionHandler
         /// </summary>
         public ConcurrentDictionary<int, Order> Orders
         {
-            get { return _orders; }
+            get {
+                return new ConcurrentDictionary<int, Order>(_closedOrders.Concat(_openOrders).
```

Comment:
note for posterity -- next PR will refactor away consumers of these public collections to use the query functions provided on the interface.